### PR TITLE
[iOS]: Allow arm64 simulator

### PR DESCRIPTION
**Pull request by 4rthurmonteiro**
We use flutter as module and we are migration our iOS app to SwiftUI and is required to run in the simulator arm64 as a valid architecture.

## Pre-launch Checklist

- [x] I made sure the project builds.
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is does not need version changes.
- [x] I updated `CHANGELOG.md` to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I rebased onto `main`.
- [x] I added new tests to check the change I am making, or this PR does not need tests.
- [x] I made sure all existing and new tests are passing.
- [x] I ran `dart format .` and committed any changes.
- [x] I ran `flutter analyze` and fixed any errors.

[Contributor Guide]: https://github.com/Baseflow/flutter-geolocator/blob/master/CONTRIBUTING.md
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: geolocator_apple/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2.3.4
+
+* Iphonesimulator arm64 as valid architecture.
```
Comment (mvanbeusekom) [inline geolocator_apple/CHANGELOG.md:3]: As for our `CHANGELOG` style we try to follow the guidelines specified by Google (as described [here](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style)).

In other words, would you mind rephrasing this to something in the lines of:

```
* Allows the ARM64 architecture as a valid IPhone simulator architecture.
```

**Review by mvanbeusekom — CHANGES_REQUESTED** [review]
Hi,

Thank you very much for this PR . I haven only one small request regarding the style of the CHANGELOG message. If you could address that I would be happy to land this PR.

File: geolocator_apple/CHANGELOG.md (hunk 1 above)
Comment (4rthurmonteiro) [reply geolocator_apple/CHANGELOG.md:3]: Thank you!

**Comment by 4rthurmonteiro** [comment]
@mvanbeusekom i updated the PR

**Review by mvanbeusekom — APPROVED** [review]
LGTM! Thanks for this contribution.

**Pull request merged**